testcases/git_diff/addition-in-middle-of-p.in.html

```diff
@@ -1,7 +1,3 @@
-<!-- This test shows broken formatting behavior: we append "adding more text here that is too long"
```

Comment:
I am not sure why this is removed in this PR.

Reply:
ditto to above

Reply:
Added back